doc/classes/Sprite2D.xml

```diff
@@ -50,6 +50,11 @@
 		</method>
 	</methods>
 	<members>
+		<member name="alignment" type="Vector2" setter="set_alignment" getter="get_alignment" default="Vector2(0, 0)">
+			Offsets the texture's drawing position by a multiple of its size.
```

Comment:
```suggestion
			The texture's drawing offset relative to sprite size.
```
Something like this. 
And also it makes sence to duplicate here a note from `centered` property (note that I did not cleaned up text below, just general suggestion).
```xml
[b]Note:[/b] For games with a pixel art aesthetic, textures may appear deformed when `relative_offset` x or y is not 0 or 1. This is caused by their position being between pixels. To prevent this, set `relative_offset` x and y to 0 or 1, or consider enabling [member ProjectSettings.rendering/2d/snap/snap_2d_vertices_to_pixel] and [member ProjectSettings.rendering/2d/snap/snap_2d_transforms_to_pixel].
```

Reply:
Updated, thanks for the suggestions.